metadata/en-US/title.txt

```diff
@@ -0,0 +1 @@
+AmneziaVPN - Multi protocol VPN client
```

Comment:
Perhaps here it is better to take the name also from google play: just `Amnezia VPN`.